graphene/types/json.py

```diff
@@ -8,7 +8,12 @@
 
 
 class JSONString(Scalar):
-    """JSON String"""
+    """
+    Allows use of a JSON String for input / output from the GraphQL schema.
+
+    Use of this type is *not recommended* as you lose the benefits of having a defined, static
```

Comment:
Is this true universally across this project?

Reply:
Lee Byron had some comments on a Map Scalar type over at 
https://github.com/graphql/graphql-spec/issues/101#issuecomment-170170967

Reply:
👍